regulations/static/regulations/js/main.js

```diff
@@ -1,9 +1,13 @@
 function makeStateful(el) {
-    const state_change_buttons = el.querySelectorAll('[data-set-state]');
+    const state_change_buttons = document.querySelectorAll('[data-set-state]');
+    const state_change_target = el.getAttribute("data-target");
```

Comment:
Rather than use a conditional down below see if it's possible to be explicit with what's being selected.

something like
```
const state_change_target = el.getAttribute("data-target");
document.querySelectorAll(`[data-set-state][data-target=${state_change_target}]`)
```
might work.

Reply:
Implemented in 69da926